src/commands/DensityProfile.cpp

```diff
@@ -55,23 +66,40 @@ For more information about chemfiles selection language, please see
                                 <stride>. The default values are 0 for <start>,
                                 the number of steps for <end> and 1 for 
                                 <stride>.
-  -s <sel>, --selection=<sel>   selection to use for the particles. This must be a
-                                selection of size 1. [default: atoms: all]
-  --profile=<profile>           type of density profile. Currently implemented are
-                                "along_axis" and "radial". [default: along_axis]
-  --axis=<axis>                 axis along which the density profile will be 
-                                computed. It should be either one of 'X','Y','Z'
+  -s <sel>, --selection=<sel>   selection to use for the particles. This must
+                                be a selection of size 1. [default: atoms: all]
+  --axis=<axis>...              computes a linear density profile along <axis>.
+                                It should be either one of 'X','Y','Z'
+                                or a vector defining the axis (e.g. 1:1:1). 
+  --radial=<axis>...            computes a radial density profile using the 
+                                distance to <axis>.
+                                It should be either one of 'X','Y','Z'
                                 or a vector defining the axis (e.g. 1:1:1). 
-                                [default: Z]
-  --origin=<coord>              coordinates for the origin of the axis.
+  --origin=<coord>              coordinates for the origin of the axis (used 
+                                only for radial profiles).
                                 [default: 0:0:0]  
   -p <n>, --points=<n>          number of points in the profile [default: 200]
   --max=<max>                   maximum distance in the profile. [default: 10]
   --min=<min>                   minimum distance in the profile.
-                                If "--profile=radial", min is 0.
-                                If "--profile=along_axis", min is -max if not precised.
+                                For radial profiles, <min> must be positive.
+                                For linear profiles, <min> is set to -<max> 
+                                if not precised.
 )";
 
+Axis axis_parse(std::string axis) {
```

Comment:
This could be a static function of the `Axis` class, and replace the string constructor:

```cpp
class Axis {
public:
    static Axis parse(std::string string);
}
```